libs/core/src/lib/decorators/element.decorator.ts

```diff
@@ -31,6 +31,11 @@ export interface IElementOptions extends IInjectOptions
      * Description displayed in Composer.
      */
     description?: string;
+
+    /**
+    * Category in Composer's Interface Assets panel.
+    */
+    category?: string;
```

Comment:
In the description of the PR, you say that the category is mandatory. Or in the decorator, it is optional.
So the category is optional or mandatory ? 
If the answer is mandatory, there is also the skeleton of the IA to modify and probably the schematics to ask for a category when creating the IA.

Reply:
Ok changes done. Category is now mandatory and we ask for it when creating the IA.